src/runtime/opengl.cpp

```diff
@@ -544,6 +560,7 @@ WEAK void delete_kernel(void *user_context, KernelInfo *kernel) {
         arg = next;
     }
     free(kernel->source);
```

Comment:
The original implementation copies the kernel source into the `kernel` struct; previously, my PR was adding an additional (leaking) copy.  I've now removed my extra copy, but this applies to the source copied by the original implementation.

Optionally, we could just not store the source.  Let me know if you want me to make that change.